src/lib/components/Post.svelte

```diff
@@ -0,0 +1,136 @@
+<script lang="ts">
+    import { PUBLIC_PREVIEW_HOST } from '$env/static/public';
+
+    import * as Card from '$lib/shadcn/components/ui/card';
+    import * as Drawer from '$lib/shadcn/components/ui/drawer';
+    import { Badge } from '$lib/shadcn/components/ui/badge';
+    import { Button } from '$lib/shadcn/components/ui/button';
+    import { Skeleton } from '$lib/shadcn/components/ui/skeleton';
+    import { Toggle } from '$lib/shadcn/components/ui/toggle';
+
+    import ArrowDown from 'lucide-svelte/icons/arrow-down';
+    import ArrowUp from 'lucide-svelte/icons/arrow-up';
+    import Share2 from 'lucide-svelte/icons/share-2';
+    import Play from 'lucide-svelte/icons/play';
+
+    import Score from './Score.svelte';
+    import ShareSheet from './ShareSheet.svelte';
+    import UserChannel from './UserChannel.svelte';
+    import Player from './Player.svelte';
+
+    const videoId = 'e0245338-7c04-4a6c-b44f-0e279a849cf5';
```

Comment:
This should be parameterized, no?

Reply:
You're right.  My intention was to parameterise it when working on the infinite scroll component, same with the user and other information about the post itself.

Reply:
The Post component has been fully parameterised.